newsfragments/1655.feature.rst

```diff
@@ -0,0 +1 @@
+Add the number of workspaces in the organization stats
```

Comment:
there is the `api` category instead of `feature`

On top of that given we changed the api, we should bump the api minor version from `1.2` to `1.3` (see `parsec/api/version.py`) and tell about this 1.3 version in the newsfragment